lib/Parse/ParsePattern.cpp

```diff
@@ -612,27 +612,18 @@ Parser::parseSingleParameterClause(ParameterContextKind paramContext,
   if (!Tok.is(tok::l_paren)) {
     // If we don't have the leading '(', complain.
     Diag<> diagID;
-    bool skipIdentifier = false;
     switch (paramContext) {
     case ParameterContextKind::Function:
     case ParameterContextKind::Operator:
       diagID = diag::func_decl_without_paren;
       break;
-    case ParameterContextKind::EnumElement:
-      diagID = diag::enum_element_decl_without_paren;
```

Comment:
@CodaFi Could you verify this is unreachable?

Reply:
Confirmed.  Good catch.